server.js

```diff
@@ -11,4 +11,8 @@ app.use(express.static('./public'));
 
 app.listen(PORT, function(){
     console.log(`listening on the port: ${PORT}`);
-});
\ No newline at end of file
+});
+
```

Comment:
Commonly, all routes are listed/set up before you call app.listen in your server.js file.